README.md

```diff
@@ -208,45 +209,43 @@ Currently, diagnosis is only available via the Dashboard.
 
 Let's take a look at the device provisioned earlier that should now be
 running the code pushed to the DebugApp. Bring up the balenaCloud Dashboard
-page and select 'Diagnostics (Experimental)' from the left-hand panel.
+page and select 'Diagnostics' from the left-hand panel.
 
-Diagnostics are split into three separate sections: health checks, diagnostics
-and Supervisor state.
+Diagnostics are split into three separate sections: Device health checks, Device diagnostics and Supervisor state.
 
 #### 2.1 Device Health Checks
 
 Select the 'Device Health Checks' tab in the Diagnostics page, and then click
-'Run checks'. A set of [health checks](https://github.com/balena-io-modules/device-diagnostics/blob/master/diagnostics.md)
+'Run checks'. This will take a couple of minutes. A set of [health checks](https://github.com/balena-io-modules/device-diagnostics/blob/master/diagnostics.md)
```

Comment:
```suggestion
'Run checks'. This may take a couple of minutes. A set of [health checks](https://github.com/balena-io-modules/device-diagnostics/blob/master/diagnostics.md)
```

Reply:
I'll update this, thanks.